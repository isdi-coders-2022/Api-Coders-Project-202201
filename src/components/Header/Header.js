import React from "react";
import styled from "styled-components";

const HeaderTheme = styled.header`
  background-color: #163344;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100px;
`;

const Title = styled.h1`
  font-family: "Sacramento", cursive;
  color: white;
  font-size: 30px;
`;
const Header = () => {
  return (
    <HeaderTheme>
      <Title className="header__title">
        Cocktails <i className="fas fa-glass-martini-alt title__icon"></i>
      </Title>
    </HeaderTheme>
  );
};

export default Header;
