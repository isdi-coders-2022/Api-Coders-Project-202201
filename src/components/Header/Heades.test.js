import { render, screen } from "@testing-library/react";
import Header from "./Header";
import reactTestRenderer from "react-test-renderer";

describe("Given a Header component", () => {
  describe("When it is rendered", () => {
    test("Then it should display a title with text 'Cocktails'", () => {
      const expectedText = "Cocktails";

      render(<Header />);

      const title = screen.queryByText(expectedText);

      expect(title).toBeInTheDocument();
    });
  });
  describe("When it's rendered", () => {
    test("Then it should be equal to the last snapshot", () => {
      const theHeader = reactTestRenderer.create(<Header />);

      expect(theHeader.toJSON()).toMatchSnapshot();
    });
  });
});
