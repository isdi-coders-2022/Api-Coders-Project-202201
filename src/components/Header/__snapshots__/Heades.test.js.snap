// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Header component When it's rendered Then it should be equal to the last snapshot 1`] = `
<header
  className="sc-bdvvtL jsna-dJ"
>
  <h1
    className="sc-gsDKAQ fHUoGR header__title"
  >
    Cocktails 
    <i
      className="fas fa-glass-martini-alt title__icon"
    />
  </h1>
</header>
`;
